packages/wallet/src/SingleAddressWallet.ts

```diff
@@ -189,8 +193,11 @@ export class SingleAddressWallet implements ObservableWallet {
         txSubmitProvider: this.txSubmitProvider,
         utxoProvider: this.utxoProvider
       },
-      { consideredOutOfSyncAfter }
+      { consideredOutOfSyncAfter },
+      contextLogger(this.#logger, 'SyncStatus')
```

Comment:
Maybe let's have those correspond to exact prop name of SingleAddressWallet?
```suggestion
      contextLogger(this.#logger, 'syncStatus')
```

Reply:
Done in [3eeba2a](https://github.com/input-output-hk/cardano-js-sdk/pull/379/commits/3eeba2a461134843d202116d39c1edec2c118116)